### PR TITLE
Add solution results page

### PR DESCRIPTION
**Pull request by sam-roberts-bjss**

### CODE REVIEW

File: app/contextCreator/createSolutionListPageContext.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export const createSolutionListPageContext = (solutions, pageTitle, pageDescription) => {
+  const context = {
+    pageTitle,
+    pageDescription,
+  };
+
+  context.solutions = solutions.map(solution => ({
```
Comment (alibombaye) [inline app/contextCreator/createSolutionListPageContext.js]: extract this to a separate function then in the context above we can just add solutions directly via this

File: app/contextCreator/createSolutionListPageContext.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export const createSolutionListPageContext = (solutions, pageTitle, pageDescription) => {
```
Comment (alibombaye) [inline app/contextCreator/createSolutionListPageContext.js]: Going to be really picky but the order of these. Would prefer title, description and then solutions

File: app/contextCreator/createSolutionListPageContext.test.js [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+        ],
+      },
+    ]
+
+    const context = createSolutionListPageContext(solutions);
+
+    expect(context).toEqual(expectedContext);
+  });
+
+  it('should create a context for the solution list page with one solution with empty capabilies', () => {
```
Comment (alibombaye) [inline app/contextCreator/createSolutionListPageContext.test.js]: typo on `capabilies`

File: app/controller.js [hunk 4]
```diff
@@ -1,65 +1,35 @@
...
+import { createSolutionPageContext } from './contextCreator/createSolutionPageContext';
+import { createSolutionListPageContext } from './contextCreator/createSolutionListPageContext';
+
+export const getSolutionFoundationListPageContext = async () => {
+  const foundationSolutionListResponse = await axios.get('http://localhost:8080/api/v1/Solutions/Foundation');
+
+  const context = createSolutionListPageContext(
+    foundationSolutionListResponse.data.solutions,
+    'Foundation Solutions - results',
+    'These Solutions meet the six Foundation Capabilities (the business needs a Solution addresses) mandated by NHS England’s GP IT Futures Operating Model. All six Capabilities must be fulfilled to achieve Foundation Solution status.',
```
Comment (alibombaye) [inline app/controller.js]: mmm actually I don't think I like these in here. I think it's more of a context thing.

File: app/contextCreator/createSolutionListPageContext.test.js (hunk 3 above)
Comment (sam-roberts-bjss) [reply app/contextCreator/createSolutionListPageContext.test.js]: Fixed

File: app/contextCreator/createSolutionListPageContext.js (hunk 1 above)
Comment (sam-roberts-bjss) [reply app/contextCreator/createSolutionListPageContext.js]: Extracted to another function.

File: app/contextCreator/createSolutionListPageContext.js (hunk 2 above)
Comment (sam-roberts-bjss) [reply app/contextCreator/createSolutionListPageContext.js]: Yeah I agree but the below comment will change this anyway

File: app/controller.js (hunk 4 above)
Comment (sam-roberts-bjss) [reply app/controller.js]: Totally agree. Doesn't look right. Moved to the context

**Pull request merged**